store/datastore.go

```diff
@@ -49,7 +49,7 @@ type DataStore interface {
 	//tenants
 	ProvisionTenant(ctx context.Context, tenantId string) error
 
-	//images
+	// images
```

Comment:
really stupid thing: why are you adding a space here? All other comments don't have spaces :)

Reply:
It's leftover from some other work, which I didn't include in this PR at the end.
I was moving things around and, by accident, I removed this line. Then I added it back, but with space.
I think it was me who put the comments without spaces in the first place, but I think they look better with spaces.
Do you want me to remove the space?